src/styles/index.scss

```diff
@@ -4,0 +1,2 @@
+@import "variables";
+@import "main";
```

Comment:
Ensure that the `main` file exists in the same directory or provide the correct path if it's located elsewhere.